cli/util/uploader.go

```diff
@@ -411,6 +412,13 @@ func updateIndexFiles(ctx *cli.Context, p *pool.Pool, containerID cid.ID, accoun
 				}
 			}
 		}
+		emptyOid := make([]byte, 32)
+		for k := 0; k < len(buffer); k += oidSize {
+			myOid := buffer[k : k+oidSize]
+			if slices.Compare(myOid, emptyOid) == 0 {
+				return fmt.Errorf("empty oid found in buffer with index %d", k/oidSize)
```

Comment:
Add details about IndexFile index. Replace `buffer` with something more appropriate so that external user may understand that something is wrong with the index file construction.